meshery-design-embed/public/embedded-design-embed1.js

```diff
@@ -1622,7 +1618,7 @@ const addStyles = () => {
    #embedded-design-a3d3f26e-4366-44e6-b211-1ba4e1a3e644 .water-mark{
```

Comment:
Why is there a specific design's UUID listed in this selector?

Reply:
I think that's the default id given by @aabidsofi19